src/metatrain/experimental/alchemical_model/utils/composition.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Tuple, Union
+
+import torch
+
+from ....utils.data.dataset import Dataset, get_atomic_types
+
+
+def calculate_composition_weights(
```

Comment:
I think this is not needed anymore?

Reply:
Unfortunately this needs to be kept for the alchemical model (notice that it changed directories), which works in a way that doesn't allow me to change things without changing the alchemical model code itself

Reply:
Okay!